test/specs/collections/Table/Table-test.js

```diff
@@ -59,7 +59,11 @@ describe('Table', () => {
 
     const headerRow = ['Name', 'Status', 'Notes']
 
-    const renderBodyRow = ({ name, status, notes }) => [name || '', status || '', notes || '']
+    const renderBodyRow = ({ name, status, notes }) => [
+      name || { key: 0 },
+      status || { key: 1 },
+      notes || { key: 2 },
```

Comment:
Since these are cells and in the event that there are multiple missing cells in the `tableData` (which there are) then the cells will all have identical props objects and therefore identical generated keys.

I've instead defaulted to objects with a key defined.